app/services/nomis/contact.rb

```diff
@@ -1,18 +1,29 @@
 module Nomis
   class Contact
-    include NonPersistedModel
+    include MemoryModel
     include Comparable
 
-    attribute :id, Integer
-    attribute :given_name
-    attribute :surname
-    attribute :date_of_birth, Date
-    attribute :gender, Hash[Symbol => String]
-    attribute :relationship_type, Hash[Symbol => String]
-    attribute :contact_type, Hash[Symbol => String]
-    attribute :approved_visitor, Boolean
-    attribute :active, Boolean
-    attribute :restrictions, Array[Restriction]
+    attribute :id, :integer
+    attribute :given_name, :string
+    attribute :middle_names, :string
+    attribute :surname, :string
+    attribute :date_of_birth, :date
+    attribute :gender
```

Comment:
@StupidCodeFactory - is a default used if a type is not declared?

Reply:
good catch will add that in